tslint.json

```diff
@@ -133,6 +133,10 @@
       }
     ],
     "curly": false,
+    "max-line-length": [
+      true,
+      150
+    ],
     "arrow-parens": false,
     "linebreak-style": [ true, "LF" ],
     "tslint.jsEnable": true
```

Comment:
This rule does not exits. please remove. throws an error
> npm run lint

Reply:
I have just ran the command above and i have not problems. 
Please try to delete your `node_modules` and install. I will also do that.

see https://palantir.github.io/tslint/rules/max-line-length/